### PR TITLE
Apply last changes in `Alert` widget.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Alert.php [hunk 1]
```diff
@@ -57,18 +58,49 @@ public function addAttributes(array $values): self
...
-    *
-    * @link https://html.spec.whatwg.org/#classes
-    */
-    public function addClass(string $value): self
+     * Adds one or more CSS classes to the existing classes of the alert component.
+     *
+     * Multiple classes can be added by passing them as separate arguments. `null` values are filtered out
+     * automatically.
+     *
+     * @param string|null ...$value One or more CSS class names to add. Pass null to skip adding a class.
```
Comment (samdark) [inline src/Alert.php:66]:
```suggestion
     * @param string|null ...$value One or more CSS class names to add. Pass `null` to skip adding a class.
```

File: src/Alert.php [hunk 2]
```diff
@@ -57,18 +58,49 @@ public function addAttributes(array $values): self
...
+        $new->cssClass = array_merge(
+            $new->cssClass,
+            array_filter($value, static fn ($v) => $v !== null)
+        );
+
+        return $new;
+    }
+
+    /**
+     * Adds a style class for the alert component.
```
Comment (samdark) [inline src/Alert.php:89]:
```suggestion
     * Adds a CSS style for the alert component.
```

File: src/Alert.php [hunk 3]
```diff
@@ -57,18 +58,49 @@ public function addAttributes(array $values): self
...
+     *
+     * @param array|string $value The style class for the alert component. If an array, the values will be separated by
```
Comment (samdark) [inline src/Alert.php:91]:
```suggestion
     * @param array|string $value The CSS style for the alert component. If an array, the values will be separated by
```

File: src/Alert.php [hunk 4]
```diff
@@ -57,18 +58,49 @@ public function addAttributes(array $values): self
...
+     * a space. If a string, it will be added as is. For example, 'color: red;'. If the value is an array, the values
+     * will be separated by a space. e.g., ['color' => 'red', 'font-weight' => 'bold'] will be rendered as
+     * 'color: red; font-weight: bold;'.
+     * @param bool $overwrite Whether to overwrite existing styles with the same name. If `false`, the new value will be
+     * appended to the existing one.
+     *
+     * @return self A new instance with the specified style class value added.
```
Comment (samdark) [inline src/Alert.php:98]:
```suggestion
     * @return self A new instance with the specified CSS style value added.
```

File: src/Alert.php [hunk 5]
```diff
@@ -111,6 +143,29 @@ public function body(string|Stringable $value, bool $encode = true): self
         return $new;
     }
 
+    /**
+     * Replaces all existing CSS classes of the alert component with the provided ones.
+     *
+     * Multiple classes can be added by passing them as separate arguments. `null` values are filtered out
+     * automatically.
+     *
+     * @param string|null ...$value One or more CSS class names to set. Pass null to skip setting a class.
```
Comment (samdark) [inline src/Alert.php:152]:
```suggestion
     * @param string|null ...$value One or more CSS class names to set. Pass `null` to skip setting a class.
```

**Review by samdark — CHANGES_REQUESTED** [review]

**Review by vjik — APPROVED** [review]

File: src/Alert.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Alert.php:66]: done.

File: src/Alert.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Alert.php:89]: done.

File: src/Alert.php (hunk 3 above)
Comment (terabytesoftw) [reply src/Alert.php:91]: done.

File: src/Alert.php (hunk 4 above)
Comment (terabytesoftw) [reply src/Alert.php:98]: done.

File: src/Alert.php (hunk 5 above)
Comment (terabytesoftw) [reply src/Alert.php:152]: done.

**Review by samdark — APPROVED** [review]

**Pull request merged**